.github/workflows/pr-test.yml

```diff
@@ -161,19 +145,20 @@ jobs:
           name: build
           path: ${{ env.BUILD_OUT_ROOT }}
 
-      - uses: technote-space/get-diff-action@v6
-        with:
-          PATTERNS: files/**/*.+(png|jpeg|jpg|gif|svg|webp)
-          ABSOLUTE: true
-          SET_ENV_NAME: GIT_DIFF_FILES
+      - name: Get changed images
+        if: ${{ env.GIT_DIFF_CONTENT }}
+        # note that we should get the absolute path of the changed images
+        run: |
+          DIFF_IMAGES=$(cat ${{ env.CHANGED_FILES }} | egrep -i "^files/.*/.*\.(png|jpeg|jpg|gif|svg|webp)$" | xargs readlink -e | tr "\n" " ")
+          echo "GIT_DIFF_IMAGES=$DIFF_IMAGES" >> $GITHUB_ENV
```

Comment:
My bad, I've ignored to export absolute path here, see testing pr (yin1999/translated-content#8):

- failed with image (unused): https://github.com/yin1999/translated-content/actions/runs/3293923349
- passed with image (used): https://github.com/yin1999/translated-content/actions/runs/3293947316/jobs/5430964574